sklego/preprocessing.py

```diff
@@ -421,6 +426,10 @@ def fit(self, X, y=None):
 
     def transform(self, X):
         """Transforms X using the fitted inverse of R. Normalizes the result if required"""
+        if type(X) is pd.core.frame.DataFrame:
+            if not self.columns_ == list(X.columns):
+                raise KeyError('Columns were not equal during fit and transform')
+        
```

Comment:
handling bad transformation of OrthogobalTransformer mentioned in #249

Reply:
This seems like something that we should either do on **all** transformers, or on None. I don't think adding this on all transformers is the way to go.

Reply:
In particular ... the end user is responsible for doing sensible things with our library. If you're gonna fit on a dataset that does not resemble what you're gonna predict on then this is something that we cannot prevent either way.

Reply:
Hmm, I agreed.

Reply:
This bit is still in the codebase. 

```
            if not self.columns_ == list(X.columns):
                raise KeyError('Columns were not equal during fit and transform')
```

Reply:
I removed these things. These images are pointing to previous one